ios/CameraView+RecordVideo.swift

```diff
@@ -6,41 +6,17 @@
 //  Copyright © 2020 mrousavy. All rights reserved.
 //
 
-import AVFoundation
-
-enum OculaExamMode: String {
-    case normal = "normal"
-    case enhancedRecording = "enhancedRecording"
-    case enhancedMetric = "enhancedMetric"
-    case enhancedAll = "enhancedAll"
+struct TorchControl {
+    var torchLength: Double
```

Comment:
You're going to hate me for this, but we should probably change torchLength to torchDuration.  Length really is more closely associated with mm, where duration is clearly associated with ms. 

```suggestion
    var torchDuration: Double
```

Reply:
it fixed.